README.md

```diff
@@ -7,37 +7,39 @@ These benchmarks are based on a similar project by
 
 _**Note**: at the moment the ExCALIBUR benchmarks are a work-in-progress._
 
+## Before you start
+
+If you are using spack for other projects, you might have to clear the cache if it exists:
+`rm -rf ~/.spack/cache`
```

Comment:
`spack clean -m` is a better way to clear the cache, instead of suggesting to `rm -rf` directories which may not even exist (for example on Acher2 and other Cray system you typically don't have anything related to spack in your home directory since it can't be mounted at all by compute nodes).

Also, I don't think the "using spack for other projects" description is accurate, you may have troubles when upgrading spack after some time, especially in upgrading past a breaking version (e.g. you were on 0.19 and now checked out a recent `development` branch which is on 0.20), they occasionally change the names of some packages, and if they stick in the cache then the resolve gets very disappointed.  And yes, this is a bit annoying.  Using Spack for other projects is irrelevant, as far as I know.